src/main/groovy/org/craftercms/cli/Main.groovy

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.cli
+
+import org.craftercms.cli.commands.*
+import picocli.CommandLine
+
+@CommandLine.Command(
+        name = 'crafter-cli', usageHelpAutoWidth = true,
+        subcommands = [CommandLine.HelpCommand, AddEnvironment, AddRemote, CreateSite, ListRemotes, SyncFrom, SyncTo])
```

Comment:
Add a version command and embed a version number. Start the version number to match the current Crafter version and get that from the master POM.